server/src/test/java/org/opensearch/search/profile/query/ConcurrentQueryProfileBreakdownTests.java

```diff
@@ -0,0 +1,93 @@
+        testMap.put(CREATE_WEIGHT + COUNT_SUFFIX, 1L);
+        testMap.put(CREATE_WEIGHT + START_TIME_SUFFIX, 1598347679188617L);
+        testMap.put(CREATE_WEIGHT + MAX_END_TIME_SUFFIX, 1598347679558960L);
+        testMap.put(CREATE_WEIGHT + MIN_START_TIME_SUFFIX, 1598347679188617L);
+        testMap.put(BUILD_SCORER, 0L);
+        testMap.put(BUILD_SCORER + COUNT_SUFFIX, 6L);
+        testMap.put(BUILD_SCORER + START_TIME_SUFFIX, 0L);
+        testMap.put(BUILD_SCORER + MAX_END_TIME_SUFFIX, 1598347688270123L);
+        testMap.put(BUILD_SCORER + MIN_START_TIME_SUFFIX, 1598347686448701L);
+        testMap.put(NEXT_DOC, 0L);
+        testMap.put(NEXT_DOC + COUNT_SUFFIX, 5L);
+        testMap.put(NEXT_DOC + START_TIME_SUFFIX, 0L);
+        testMap.put(NEXT_DOC + MAX_END_TIME_SUFFIX, 1598347688298671L);
+        testMap.put(NEXT_DOC + MIN_START_TIME_SUFFIX, 1598347688110288L);
+        testMap.put(ADVANCE, 0L);
+        testMap.put(ADVANCE + COUNT_SUFFIX, 3L);
+        testMap.put(ADVANCE + START_TIME_SUFFIX, 0L);
+        testMap.put(ADVANCE + MAX_END_TIME_SUFFIX, 1598347688280739L);
+        testMap.put(ADVANCE + MIN_START_TIME_SUFFIX, 1598347687991984L);
+        testMap.put(MATCH, 0L);
+        testMap.put(MATCH + COUNT_SUFFIX, 0L);
+        testMap.put(MATCH + START_TIME_SUFFIX, 0L);
+        testMap.put(MATCH + MAX_END_TIME_SUFFIX, 0L);
+        testMap.put(MATCH + MIN_START_TIME_SUFFIX, 0L);
+        testMap.put(SCORE, 0L);
+        testMap.put(SCORE + COUNT_SUFFIX, 5L);
+        testMap.put(SCORE + START_TIME_SUFFIX, 0L);
+        testMap.put(SCORE + MAX_END_TIME_SUFFIX, 1598347688282743L);
+        testMap.put(SCORE + MIN_START_TIME_SUFFIX, 1598347688018500L);
+        testMap.put(SHALLOW_ADVANCE, 0L);
+        testMap.put(SHALLOW_ADVANCE + COUNT_SUFFIX, 0L);
+        testMap.put(SHALLOW_ADVANCE + START_TIME_SUFFIX, 0L);
+        testMap.put(SHALLOW_ADVANCE + MAX_END_TIME_SUFFIX, 0L);
+        testMap.put(SHALLOW_ADVANCE + MIN_START_TIME_SUFFIX, 0L);
+        testMap.put(COMPUTE_MAX_SCORE, 0L);
+        testMap.put(COMPUTE_MAX_SCORE + COUNT_SUFFIX, 0L);
+        testMap.put(COMPUTE_MAX_SCORE + START_TIME_SUFFIX, 0L);
+        testMap.put(COMPUTE_MAX_SCORE + MAX_END_TIME_SUFFIX, 0L);
+        testMap.put(COMPUTE_MAX_SCORE + MIN_START_TIME_SUFFIX, 0L);
+        testMap.put(SET_MIN_COMPETITIVE_SCORE, 0L);
+        testMap.put(SET_MIN_COMPETITIVE_SCORE + COUNT_SUFFIX, 0L);
+        testMap.put(SET_MIN_COMPETITIVE_SCORE + START_TIME_SUFFIX, 0L);
+        testMap.put(SET_MIN_COMPETITIVE_SCORE + MAX_END_TIME_SUFFIX, 0L);
+        testMap.put(SET_MIN_COMPETITIVE_SCORE + MIN_START_TIME_SUFFIX, 0L);
+        ConcurrentQueryProfileBreakdown profileBreakdown = new ConcurrentQueryProfileBreakdown();
+        Map<String, Map<String, Long>> sliceLevelBreakdown = new HashMap<>();
+        sliceLevelBreakdown.put("testCollector", testMap);
+        Map<String, Long> breakdownMap = profileBreakdown.buildFinalBreakdownMap(sliceLevelBreakdown);
```

Comment:
this needs to be changed to call the new method `buildQueryBreakdownMap`. Also can you add multiple slices breakdown in the sliceLevelBreakdown with non-zero values and then verify the results in `queryBreakdownMap`.

Reply:
Have added more UTs for this in latest commit